inc/infoutils.php

```diff
@@ -444,9 +444,10 @@ function dbglog($msg,$header=''){
  * Log accesses to deprecated fucntions to the debug log
  *
  * @param string $alternative The function or method that should be used instead
+ * @param string|null $deprecatedThing What is deprecated if not the current method
  * @triggers INFO_DEPRECATION_LOG
  */
-function dbg_deprecated($alternative = '') {
+function dbg_deprecated($alternative = '', $deprecatedThing = null) {
```

Comment:
How about making that second parameter an integer that indicates how far back in the stacktrace the call happend (default: `$backtrack=1`)? For the usecase here, passing a 2 should work.

Then a couple of lines further down `$backtrace = array_splice($backtrace, $backtrack);` instead of the `array_shift` should work.

Reply:
Ah no. Nevermind, I just realize that the problem is that the stacktrace points to a function but we want a property.